shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -26,6 +26,10 @@ case class GlutenNumaBindingInfo(
 
 class GlutenConfig(conf: SQLConf) extends Logging {
 
+  // With this setting, row based input will be used in the fallback case of data source v1.
+  // TODO: remove this setting if vanilla spark's vectorized input is supported.
+  conf.setConfString("spark.sql.parquet.enableVectorizedReader", "false")
+
```

Comment:
what happens if we use columnar input? Will it add a columnarToRow, then RowTo ArrowColumnar?

Reply:
At least for velox,` we have no support to transform vanilla spark columnar input to velox columnar input. So if we do use vanilla columnar input, an exception will be thrown to interrupt the execution. cc @rui-mo.
BTW, we found setting the configuration here can cause some issues, maybe we have to tell user to set it manually.